tests/inputs/deprecated/deprecated.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+// Some documentation about the Test message.
+message Test {
+    // Some documentation about the value.
+    option deprecated = true;
+    int32 v = 1 [deprecated=true];
```

Comment:
Is `int32 v = 1 [deprecated=true]` redundant with the `deprecated_field` test?

Reply:
It was done specificaly to verify that we handle the case where the message and the field deprecation explicitly.